### ISSUE TITLE
Hand Gloves Detection using DL

### ISSUE THREAD

**Issue by abhisheks008** [World-of-ML/DL-Simplified#230]
### Deep Learning Simplified Repository (Proposing new issue)
:red_circle: **Project Title** : Hand Gloves Detection</br>
:red_circle: **Aim** : Create a DL model which will detect the gloves.</br>
:red_circle: **Dataset** : https://www.kaggle.com/datasets/dataclusterlabs/gloves-dataset-covid-safety-wear</br>
:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- GitHub Profile Link : 
- Email ID :
- Participant ID (if applicable):
- Approach for this Project :
- What is your participant role? (Mention the Open Source program)

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by GitArya27** [comment]
Happy to contribute in this project!

Full name : Arya Singh
GitHub Profile Link : https://github.com/GitArya27
Email ID : [singharya2792@gmail.com](mailto:singharya2792@gmail.com)
Participant ID (if applicable):
Approach for this Project : I will be using Transfer Learning with Pre-trained CNN, Haar Cascade Classifier with OpenCV, and Simple Deep Neural Network (DNN) to implement the models.
What is your participant role? - SSOC Participant

Kindly assign this issue to me, thank you.

**Comment by Sagar201003** [comment]
I can work on this project . As I have made a deep learning model , similar to this one , which is face mask detection .

**Comment by Sagar201003** [comment]
Full name : Sagar Santoshkumar Shukla
GitHub Profile Link :https://github.com/Sagar201003
Email ID : shuklasagar201003@gmail.com
Participant ID (if applicable):
Approach for this Project :
What is your participant role? (Mention the Open Source program) -  SSOC contributor

**Comment by abhisheks008** [comment]
You can work on one issue at a time only.
@riiyaa24

**Comment by achrekarom12** [comment]
Full name : Om Achrekar
GitHub Profile Link : achrekarom12
Email ID : [achrekarom@gmail.com](mailto:achrekarom@gmail.com)
Approach for this Project : Using various Image detection models such as VGG19, Inception and Other suitable image detection model
What is your participant role? SSoC '23 Contributor

**Comment by abhisheks008** [comment]
Issue assigned to you @achrekarom12

**Comment by achrekarom12** [comment]
Hey @abhisheks008 , 
Just letting you know, the dataset you've attached is incomplete.
It only have images of hands with gloves on it. So I found another dataset with only human hand. They are from same authors. 

This dataset I found which complements the dataset you have given:
https://www.kaggle.com/datasets/dataclusterlabs/palm-and-gloves-dataset

I will include this too in README file.
